### PR TITLE
implement goerli balanceTracker and feeDisburser deployment

### PR DESCRIPTION
**Pull request by dapperscene6**
Implements balanceTracker and feeDisburser deployment and an upgrade feeDisburser proxy to the deploy feeDisburser implementation

### CODE REVIEW

File: goerli/2023-08-23-deploy-revshare/.env [hunk 1]
```diff
@@ -0,0 +1,34 @@
+OP_COMMIT=3580bf1b41d80fcb2b895d5610836bfad27fc989
+BASE_CONTRACTS_COMMIT=de5d9417c144a78743d2e10c4e2b4effbb2d1acd
+
+# BalanceTracker configuration
+BALANCE_TRACKER_DEPLOYER=0xf2faee04500bcd6797cb8545f9edd7c933b99835
+PROFIT_WALLET=0x5a822ea15764a6090b86b1eabffc051cec99afe9
+OUTPUT_PROPOSER=0x4f28beb40635093b72def43d0ad801cdd8e0e6fe
+BATCH_SENDER=0x73b4168Cc87F35Cc239200A20Eb841CdeD23493B
+OUTPUT_PROPOSER_TARGET_BALANCE=200000000000000000000
```
Comment (anikaraghu) [inline goerli/2023-08-23-deploy-revshare/.env:9]: 200 ETH

File: goerli/2023-08-23-deploy-revshare/.env [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+BATCH_SENDER_TARGET_BALANCE=1000000000000000000000
```
Comment (anikaraghu) [inline goerli/2023-08-23-deploy-revshare/.env:10]: 1000 ETH

File: goerli/2023-08-23-deploy-revshare/.env [hunk 3]
```diff
@@ -0,0 +1,34 @@
+OP_COMMIT=3580bf1b41d80fcb2b895d5610836bfad27fc989
+BASE_CONTRACTS_COMMIT=de5d9417c144a78743d2e10c4e2b4effbb2d1acd
+
+# BalanceTracker configuration
+BALANCE_TRACKER_DEPLOYER=0xf2faee04500bcd6797cb8545f9edd7c933b99835
```
Comment (anikaraghu) [inline goerli/2023-08-23-deploy-revshare/.env:5]: confirmed this is funded 👍 https://goerli.etherscan.io/address/0xf2faee04500bcd6797cb8545f9edd7c933b99835

File: goerli/2023-08-23-deploy-revshare/.env [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+PROFIT_WALLET=0x5a822ea15764a6090b86b1eabffc051cec99afe9
```
Comment (anikaraghu) [inline goerli/2023-08-23-deploy-revshare/.env:6]: confirmed this is unused https://goerli.basescan.org/address/0x5a822ea15764a6090b86b1eabffc051cec99afe9

File: goerli/2023-08-23-deploy-revshare/.env [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+OUTPUT_PROPOSER=0x4f28beb40635093b72def43d0ad801cdd8e0e6fe
```
Comment (anikaraghu) [inline goerli/2023-08-23-deploy-revshare/.env:7]: confirmed this is proposer: https://goerli.etherscan.io/address/0x4f28beb40635093b72def43d0ad801cdd8e0e6fe 
nit: maybe appending these addresses with `L1_` or `L2_` could make reviewing easier!

File: goerli/2023-08-23-deploy-revshare/.env [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+BATCH_SENDER=0x73b4168Cc87F35Cc239200A20Eb841CdeD23493B
```
Comment (anikaraghu) [inline goerli/2023-08-23-deploy-revshare/.env:8]: confirmed batcher: https://goerli.etherscan.io/address/0x73b4168Cc87F35Cc239200A20Eb841CdeD23493B

File: goerli/2023-08-23-deploy-revshare/.env [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+OUTPUT_PROPOSER_TARGET_BALANCE=200000000000000000000
+BATCH_SENDER_TARGET_BALANCE=1000000000000000000000
+BALANCE_TRACKER_ADMIN=0x4C35Ca57616E0d5fD808574772f632D8dA4eadCa
```
Comment (anikaraghu) [inline goerli/2023-08-23-deploy-revshare/.env:11]: confirmed this is our safe https://goerli.etherscan.io/address/0x4C35Ca57616E0d5fD808574772f632D8dA4eadCa

File: goerli/2023-08-23-deploy-revshare/.env [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+BALANCE_TRACKER_SALT=BalanceTrackerProxyV2
+BALANCE_TRACKER_CHAIN_ID=5
+
+# FeeDisburser configuration
+FEE_DISBURSER_DEPLOYER=0xf2faee04500bcd6797cb8545f9edd7c933b99835
+OPTIMISM_WALLET=0x5a822ea15764a6090b86b1eabffc051cec99afe9
+FEE_DISBURSEMENT_INTERVAL=604800 # 60 seconds * 60 minutes * 24 hours * 7 days
+FEE_DISBURSER_ADMIN=0x2eD486761dcF287E7b79E526B0d3fC2349834a66
```
Comment (anikaraghu) [inline goerli/2023-08-23-deploy-revshare/.env:19]: Confirmed this is our safe https://goerli.basescan.org/address/0x2eD486761dcF287E7b79E526B0d3fC2349834a66

File: goerli/2023-08-23-deploy-revshare/.env [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+OUTPUT_PROPOSER=0x4f28beb40635093b72def43d0ad801cdd8e0e6fe
+BATCH_SENDER=0x73b4168Cc87F35Cc239200A20Eb841CdeD23493B
+OUTPUT_PROPOSER_TARGET_BALANCE=200000000000000000000
+BATCH_SENDER_TARGET_BALANCE=1000000000000000000000
+BALANCE_TRACKER_ADMIN=0x4C35Ca57616E0d5fD808574772f632D8dA4eadCa
+BALANCE_TRACKER_SALT=BalanceTrackerProxyV2
+BALANCE_TRACKER_CHAIN_ID=5
+
+# FeeDisburser configuration
+FEE_DISBURSER_DEPLOYER=0xf2faee04500bcd6797cb8545f9edd7c933b99835
```
Comment (anikaraghu) [inline goerli/2023-08-23-deploy-revshare/.env:16]: confirmed this is funded 👍 https://goerli.basescan.org/address/0xf2faee04500bcd6797cb8545f9edd7c933b99835

File: goerli/2023-08-23-deploy-revshare/.env [hunk 10]
```diff
@@ -0,0 +1,34 @@
...
+OPTIMISM_WALLET=0x5a822ea15764a6090b86b1eabffc051cec99afe9
```
Comment (anikaraghu) [inline goerli/2023-08-23-deploy-revshare/.env:17]: How do we confirm this?

**Review by anikaraghu — APPROVED** [review]
lgtm!

File: goerli/2023-08-23-deploy-revshare/.env (hunk 10 above)
Comment (dapperscene6) [reply goerli/2023-08-23-deploy-revshare/.env:17]: It's a Coinbase custodied key.

**Pull request merged**